packages/ui/src/icons/chrome-icon.tsx

```diff
@@ -0,0 +1,228 @@
+import * as React from 'react';
```

Comment:
Debating whether this belongs in ui. Is it domain-agnostic? 🤔 Maybe, I guess. But could also be seen as specific to this feature, thus belonging in `app`.

Reply:
Yeah, I see the argument for that! I've kind of liked simply throwing all new icons into UI, it just lets us work in one place. Would be curious to hear @aulneau 's opinion.

Reply:
I am generally in favor of all icons being in the ui lib, because they truly are generic, typically at least. It helps force consistency for iconography. I don't think we lose anything by adding it.